README.md

```diff
@@ -152,7 +152,7 @@ There are two common ways to verify a downloaded release, depending if `gh` is a
   $ cosign verify-blob-attestation --bundle cli-cli-attestation-3120304.sigstore.json \
         --new-bundle-format \
         --certificate-oidc-issuer="https://token.actions.githubusercontent.com" \
-        --certificate-identity-regexp="^https://github.com/cli/cli/.github/workflows/deployment.yml@refs/heads/trunk$" \
+        --certificate-identity-regexp='^https://github\.com/cli/cli/\.github/workflows/deployment\.yml@refs/heads/trunk$' \
```

Comment:
Shouldn't slashes also be escaped?

Reply:
smart.

Reply:
I really don't know what the intention is here, this is under the purview of @cli/package-security, it does seem like `--certificate-identity` could be passed directly.

Reply:
You can pass `--cert-identity` too: https://github.com/cli/cli/blob/trunk/pkg/cmd/attestation/verify/verify.go#L241

The `--cert-identity-regex` uses Go style regex 👍